### DISCUSSION TITLE
log applicable 'virt-install' command

### DISCUSSION THREAD

**Discussion by paul-hammant** [cockpit-project/cockpit#20793] [Q&A]
Page: Create VM

Cockpit would be even better if each UI-creation of a VM logged the equivalent `virt-install` command used. This would serve as education for users, such that they could transition to scripted creation of VMs.  There logs are created for "`New machine qemu-NUMBER-NAME`" but none of the command used. I realize that cockpit does not shell out to `virt-install` specifically, but the equivalent command with arg would be great.

Same would be true for podman containers :)

**Comment by martinpitt** [comment]
Sorry @paul-hammant for the back and forth, I thought something wrong at first. It's https://github.com/cockpit-project/cockpit-machines/issues/1736 now.
